streamer/src/nonblocking/quic.rs

```diff
@@ -1270,10 +1270,22 @@ async fn handle_chunks(
     let bytes_sent = accum.meta.size;
     let chunks_sent = accum.chunks.len();
 
-    if let Err(err) = packet_sender.send(accum.clone()).await {
+    if let Err(err) = packet_sender.try_send(accum.clone()) {
         stats
             .total_handle_chunk_to_packet_batcher_send_err
             .fetch_add(1, Ordering::Relaxed);
+        match err {
+            TrySendError::Full(_) => {
+                stats
+                    .total_handle_chunk_to_packet_batcher_send_full_err
+                    .fetch_add(1, Ordering::Relaxed);
+            }
+            TrySendError::Disconnected(_) => {
+                stats
```

Comment:
When would the channel be disconnected on the other side and end up getting reconnected again?

Reply:
no. It probably can only happen during the service shutdown when the coalescer exited.

Reply:
in this case why are we logging stats instead of exiting?

Reply:
1. This is not an issue caused by this PR -- we had similar condition even with async channel being disconnected, this just made the handling more explicit of error conditions as a result of the channel full condition.
2. We do not want to exit (panic). Normally the exit condition in the caller will be signaled the QUIC streamer server will be terminated there.

Reply:
Ok I understand that this is the intended logic, but if there is a panic in the thread holding the other end of the channel, would we not become stuck in a loop here instead of exiting?

Reply:
How can the other end panic? I think current the other end can exit only during service (valid exit) shutdown currently. As mentioned when validator exit, the Exit variable is signaled, server started by run_server (https://github.com/anza-xyz/agave/blob/master/streamer/src/nonblocking/quic.rs#L282) will be shutdown triggering closing all connections. 

Again, there is no logic change here, before this PR we have the same 'problem'  if any.

Reply:
I understand there is no logic change, what I am trying to understand is whether this logic was even appropriate in the first place. Can you follow up on this? If it is indeed something that never really happens, maybe we can get rid of metric associated with this event too?

Reply:
The condition can be triggered, but it won't cause dead loop. I think the metrics is still useful. And it will be more so when we get support to dynamically switch off the native TPU to use vortexor.

Reply:
Thank you for checking!